addons/l10n_gcc_pos/static/src/overrides/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -30,7 +30,7 @@
         </xpath>
         <xpath expr="//span[@t-esc='props.formatCurrency(props.data.amount_total)']/.." position="after">
             <div t-if="props.data.is_gcc_country" class="pos-receipt-amount pos-receipt-amount-arabic" t-translation="off">
-                TOTAL / الإجمالي
+                TOTAL / اﻹجمالي
```

Comment:
?

Reply:
Just a different way of writing it
It was irritating me